app/models/event.rb

```diff
@@ -142,4 +143,9 @@ def calc_duration #This is also used in timecards; it should be extracted out
   def trim_id_code
     self.id_code = self.id_code.split[0].downcase
   end
+
+  def expired?
```

Comment:
I would expect a method ending in `?` to return a boolean.

Reply:
Yes, it will return either true or false. If True it will not validate uniqueness, if false it will.

Reply:
I think I like the style you used in the notification model. Won't this return either `nil` or a collection of Events ? The `empty?` helps.

Reply:
I'll go ahead and update that. It makes sense for the methods to match anyway.

Reply:
Done.